app/views/transferSummary.scala.html

```diff
@@ -15,24 +15,24 @@ <h1 class="govuk-heading-xl">@Messages("transferSummary.title")</h1>
                     <dt class="govuk-summary-list__key">
                         @Messages("transferSummary.seriesReference")
                     </dt>
-                    <dd class="govuk-summary-list__value summary-list__right-column">
-
+                    <dd class="govuk-summary-list__value">
+                        @summary.seriesCode
                     </dd>
                 </div>
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">
                         @Messages("transferSummary.transferringBody")
                     </dt>
-                    <dd class="govuk-summary-list__value summary-list__right-column">
-
+                    <dd class="govuk-summary-list__value">
+                        @summary.transferringBody
                     </dd>
                 </div>
                 <div class="govuk-summary-list__row">
                     <dt class="govuk-summary-list__key">
                         @Messages("transferSummary.filesUploadedForTransfer")
                     </dt>
-                    <dd class="govuk-summary-list__value summary-list__right-column">
-
+                    <dd class="govuk-summary-list__value">
+                        @summary.totalFiles @if(summary.totalFiles == 1) {file} else {files} uploaded
```

Comment:
This is cool! I didn't know you could do the messages with arguments like that! :1st_place_medal:

Reply:
Ah yeah, thanks, got there through quite a bit of trial and error. I just thought, "since the HTML content is a string, you might be able to put these together and have them concatenate"